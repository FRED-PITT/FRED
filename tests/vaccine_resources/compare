#!/bin/bash
echo diff -b OUT.TEST/RUN1/out.csv OUT.RT/RUN1/out.csv
diff -b OUT.TEST/RUN1/out.csv OUT.RT/RUN1/out.csv

echo diff -b OUT.TEST/RUN2/out.csv OUT.RT/RUN2/out.csv
diff -b OUT.TEST/RUN2/out.csv OUT.RT/RUN2/out.csv

echo cmp -b OUT.TEST/RUN1/health_records.txt OUT.RT/RUN1/health_records.txt
cmp -b OUT.TEST/RUN1/health_records.txt OUT.RT/RUN1/health_records.txt

echo cmp -b OUT.TEST/RUN2/health_records.txt OUT.RT/RUN2/health_records.txt
cmp -b OUT.TEST/RUN2/health_records.txt OUT.RT/RUN2/health_records.txt


