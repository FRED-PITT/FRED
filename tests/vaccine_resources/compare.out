diff -b OUT.TEST/RUN1/out.csv OUT.RT/RUN1/out.csv
diff -b OUT.TEST/RUN2/out.csv OUT.RT/RUN2/out.csv
cmp -b OUT.TEST/RUN1/health_records.txt OUT.RT/RUN1/health_records.txt
cmp -b OUT.TEST/RUN2/health_records.txt OUT.RT/RUN2/health_records.txt
